### PR TITLE
Specify missing destination parent folder are created. Specify -Force…

### PR DESCRIPTION
**Pull request by mikeburek**
… also gets through reading Tag errors.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/Storage.Management/help/Copy-AzStorageBlob.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Copy-AzStorageBlob -AbsoluteUri <String> -DestContainer <String> -DestBlob <Stri
 ```
 
 ## DESCRIPTION
-The **Copy-AzStorageBlob** cmdlet copies a blob synchronously, currently only support block blob.
+The **Copy-AzStorageBlob** cmdlet copies a blob synchronously, currently only support block blob. This will create any missing parent folders in the DestBlob parameter.
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Copy-AzStorageBlob.md:40]: This is not correct.
For Blob, the Directory are virtual directory (not really exist). So Powershell actually won't create the parent directory, but will just copy the destination blob.

File: src/Storage/Storage.Management/help/Copy-AzStorageBlob.md [hunk 2]
```diff
@@ -255,7 +255,9 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Force to overwrite the existing blob or file
+Answers 'yes' to any confirmation prompts.
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Copy-AzStorageBlob.md:258]: This is not right.
`-Force` will only suspress ShouldContinue, but will not suppress other prompt like ShouldProcess.

File: src/Storage/Storage.Management/help/Copy-AzStorageBlob.md [hunk 3]
```diff
@@ -255,7 +255,9 @@ Accept wildcard characters: False
...
+Force 'yes' to overwrite the existing blob or file.
+Force 'yes' to "Can't get source blob Tags, so source blob tags won't be copied to dest blob. Do you want to continue the blob copy?" when there is an HTML status error 401, 403, or 404 when reading Tags from the blob or file.
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Copy-AzStorageBlob.md:260]: With `-Force`, we will not input 'yes', but just skip the prompt.
And `-Force` will suspress all error codes in get blob tag, but not only 401,403 or 404.

So a better description might be:

> Force to overwrite the existing blob or file.
> Force to continue the copy operation, even can't get blob tag from source blob.

**Comment by vidai-msft** [comment]
@mikeburek Could you please update this PR per the suggestion? Thanks for your contribution!

**Comment by YanaXu** [comment]
Hi @mikeburek , this PR has been idle for more than 2 months. Could you please update it according to the comments? Or we'll close a week later.

**Comment by YanaXu** [comment]
Hi @mikeburek , I'll close this PR for now. Feel free to create a new one for the update.

**Pull request closed without merging**